internal/api/namespace.go

```diff
@@ -30,23 +29,31 @@ func namespaceToProto(ns namespace.Namespace) *stencilv1beta1.Namespace {
 
 // CreateNamespace handler for creating namespace
 func (a *API) CreateNamespace(ctx context.Context, in *stencilv1beta1.CreateNamespaceRequest) (*stencilv1beta1.CreateNamespaceResponse, error) {
+	endFunc := a.newrelic.StartGenericSegment(ctx, "CreateNamespace")
+	defer endFunc()
```

Comment:
Prefer one liner here and in all the occurrences 
`defer a.newrelic.StartGenericSegment(ctx, "CreateNamespace")`